scraper.go

```diff
@@ -210,56 +212,58 @@ func (s *Scraper) Run() {
 }
 
 func (s *Scraper) runSource(src *LogSource) {
-	raw, err := os.Open(src.Filename)
-	if err != nil {
-		if src.errors.tick(commonErrorFileOpen) {
-			s.logMetaf("Error opening log file: %v", err)
+	if src != nil {
```

Comment:
Rather than nest the entire method in this if statement, it would be neater to just return or throw an error here if src == nil